src/game-id.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/prefer-default-export */
+// API endpoint /games/
+// Allowed actions: POST
+// Mandatory parameters for POST: name (string)
+
+// Create a new game or get an existing one at localStorage
```

Comment:
i Can't post a score.
Kindly fix this error related with your submit button:
![Screen Shot 2021-09-02 at 6 51 10](https://user-images.githubusercontent.com/22258274/131838794-d52136fb-d7b8-40d9-9172-44db2c29a0c6.png)